File: 백준 Java/Step4 while문/src/Pro4_3.java
import java.io.*;

public class Pro4_3 {
	public static void main(String[] args) throws IOException{
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		int n = Integer.parseInt(br.readLine());
		int num = n;
		int i = 0;
		while (true) {
			int m = 10*(num%10) + (num/10+num%10)%10;
			i+=1;
			num = m;
			if(m==n) {
				break;
			}
		}
		System.out.println(i);
	}
}

File: 백준 Python/Step4 while문/pro4_3.py
import sys
n=int(sys.stdin.readline())
num=n
i=0
while True :
    a=num//10
    b=num%10
    m=10*b+(a+b)%10
    i+=1
    num=m
    if m==n :
        break
print(i)
